### PR TITLE
v3(unfinished yet)

### LINKED ISSUE #206: Colored box-shadows
Colored box-shadows aren't really a thing in Tailwind and I don't know if they'll ever be. [Tailwind developers have pretty clearly stated that they don't plan to add it for now](https://github.com/tailwindlabs/tailwindcss/issues/654#issuecomment-473572303), but would be cool if Windi could.

Alternatively(or in addition), auto-inferred support for box-shadow colors would be sweet. Something like `shadow-xl-${color}`/`shadow-xl-hex-${hex}`.

### PR DESCRIPTION
**Pull request by voorjaar**
- [x]  Attributify mode
- [x] Windi Lang 80% (not enable for now)
- [x] New Colors dark & light
- [x] New Utilities shadow-${color} 
- [x] Alias config
- [x] Bug fixes

fixes #206 #262 #246 #245 #243 #269

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "windicss",
-  "version": "2.5.14",
+  "version": "3.0.0-beta1",
```
Comment (antfu) [inline package.json:3]:
```suggestion
  "version": "3.0.0-beta.1",
```

**Comment by antfu** [comment]
@knightly-bot build this

**Comment by voorjaar** [comment]
> @knightly-bot build this

wow. it's an amazing operation.

**Comment by voorjaar** [comment]
@knightly-bot build this

**Comment by antfu** [comment]
@knightly-bot release now

**Pull request merged**